src/bundle/test-transformer.ts

```diff
@@ -34,6 +38,15 @@ export const setupTransformHook = (opts: { removeNonJsImports?: boolean } = {}):
 
                 if (extname && !extname.match(JS_EXTENSION_RE)) {
                     path.remove();
+                    return;
+                }
+
+                try {
+                    requireModuleSync(path.node.source.value);
+                } catch (err) {
+                    if (shouldIgnoreImportFromStyleFile(err as Error)) {
+                        path.remove();
+                    }
```

Comment:
I think function should be named "shouldIgnoreImportFromStyleFile" only if we are sure this is style file.

"shouldIgnoreImportFileError" or "shouldIgnoreImportError" is more suitable, in my opinion, because we check "is this style file" inside of the function